src/plugins/eqht-css/eqht-css-en.hbs

```diff
@@ -0,0 +1,425 @@
+						<li>test</li>
+						<li>test</li>
+					</ul>
+				</div>
+			</section>
+		</div>
+
+		<div class="col-sm-6 col-md-3">
+			<section class="panel panel-default hght-inhrt-css">
+				<div class="panel-heading">
+					<h3 class="panel-title">Column 4</h3>
+				</div>
+				<div class="panel-body">
+					<p>Column 4</p>
+					<ul>
+						<li>test</li>
+						<li>test</li>
+						<li>test</li>
+						<li>test</li>
+						<li>test</li>
+					</ul>
+				</div>
+			</section>
+		</div>
+	</div>
+	<section>
+		<h3>Code</h3>
+		<pre><code>&lt;section&gt;
+	&lt;h2&gt;Nested Example&lt;/h2&gt;
+
+	&lt;div class="row wb-eqht-grd"&gt;
+		&lt;div class="col-sm-6 col-md-3"&gt;
+			&lt;section class="panel panel-default hght-inhrt-css"&gt;
+				&lt;div class="panel-heading"&gt;
+					&lt;h3 class="panel-title"&gt;Column 1&lt;/h3&gt;
+				&lt;/div&gt;
+				&lt;div class="panel-body"&gt;
+					&lt;p&gt;Column 1&lt;/p&gt;
+					...
+				&lt;/div&gt;
+			&lt;/section&gt;
+		&lt;/div&gt;
+
+		&lt;div class="col-sm-6 col-md-3"&gt;
+			&lt;section class="panel panel-default hght-inhrt-css"&gt;
+				&lt;div class="panel-heading"&gt;
+					&lt;h3 class="panel-title"&gt;Column 2&lt;/h3&gt;
+				&lt;/div&gt;
+				&lt;div class="panel-body"&gt;
+					&lt;p&gt;Column 2&lt;/p&gt;
+					...
+				&lt;/div&gt;
+			&lt;/section&gt;
+		&lt;/div&gt;
+
+		&lt;div class="col-sm-6 col-md-3"&gt;
+			&lt;section class="panel panel-default hght-inhrt-css"&gt;
+				&lt;div class="panel-heading"&gt;
+					&lt;h3 class="panel-title"&gt;Column 3&lt;/h3&gt;
+				&lt;/div&gt;
+				&lt;div class="panel-body"&gt;
+					&lt;p&gt;Column 3&lt;/p&gt;
+					...
+				&lt;/div&gt;
+			&lt;/section&gt;
+		&lt;/div&gt;
+
+		&lt;div class="col-sm-6 col-md-3"&gt;
+			&lt;section class="panel panel-default hght-inhrt-css"&gt;
+				&lt;div class="panel-heading"&gt;
+					&lt;h3 class="panel-title"&gt;Column 4&lt;/h3&gt;
+				&lt;/div&gt;
+				&lt;div class="panel-body"&gt;
+					&lt;p&gt;Column 4&lt;/p&gt;
+					...
+				&lt;/div&gt;
+			&lt;/section&gt;
+		&lt;/div&gt;
+	&lt;/div&gt;
+&lt;/section&gt;</code></pre>
+	</section>
+</section>
+
+<section>
+	<h2>Targeting a deeper nested elements</h2>
+	<p>The following example do the same as if the <code>hght-inhrt-css</code> class was applied on the element to be equalize as well as all the parents (up to the child of wb-eqht-grd) whereas <code>eqht-trgt-css</code> is only applied to the element to equalize (nothing needs to be done about the parents).</p>
+	<div class="row wb-eqht-grd">
+		<div class="col-sm-6 col-md-4">
+			<div class="well well-lg hght-inhrt-css">
+				<section class="panel panel-default eqht-trgt-css">
+					<div class="panel-heading">
+						<h3 class="panel-title">Column 1</h3>
+					</div>
+					<div class="panel-body">
+						<p>Column 1</p>
+						<ul>
+							<li>test</li>
+							<li>test</li>
+							<li>test</li>
+							<li>test</li>
+							<li>test</li>
+						</ul>
+					</div>
+				</section>
+			</div>
+		</div>
```

Comment:
```
<div class="row wb-eqht-grd">
		<div class="col-sm-6 col-md-4">
			<div>
			<div class="well well-lg hght-inhrt-css">
				<div>
				<section class="panel panel-default eqht-trgt-css">
					<div class="panel-heading">
						<h3 class="panel-title">Column 1</h3>
					</div>
					<div class="panel-body">
						<p>Column 1</p>
						<ul>
							<li>test</li>
							<li>test</li>
							<li>test</li>
							<li>test</li>
							<li>test</li>
						</ul>
					</div>
				</section>
				</div>
			</div>
			</div>
		</div>
```

Reply:
If this is still relevant, will need more details